laravel/app/Http/Controllers/PagesController.php

```diff
@@ -352,9 +352,4 @@ public function getFeedback($date)
         return view('pages.feedback', compact('data_date'));
     }
 
-    public function test()
-    {
-        $parser = new EvaluationParser(date('omd', strtotime('today')), true);
-        return 'test1';
-    }
```

Comment:
I think there's actually a route that maps to this. I'm totally okay with deleting it here (even though it's technically out of scope for the PR), but make sure you delete the route as well.